src/components/DetailedRoute/DetailedRoute.tsx

```diff
@@ -70,4 +73,4 @@ const DetailedRoute: FC<DetailedRouteProps> = ({ userId }) => {
   )
```

Comment:
This looks great! Way to go. I think the next step is just to make this dynamic!